dot/parachain/prospective-parachains/prospective_parachains.go

```diff
@@ -43,6 +44,16 @@ type view struct {
 	implicitView   backing.ImplicitView
 }
 
+func newView() *view {
+	return &view{
+		perRelayParent: make(map[common.Hash]*relayParentData),
+		activeLeaves:   make(map[common.Hash]bool),
+
+		// TODO: currently there's no implementation for ImplicitView
```

Comment:
is there an issue for this?

Reply:
Yup, https://github.com/ChainSafe/gossamer/issues/4634